easybuild/easyblocks/generic/configuremake.py

```diff
@@ -350,13 +351,13 @@ def test_step(self):
         """
 
         test_cmd = self.cfg.get('test_cmd') or DEFAULT_TEST_CMD
-        if self.cfg['runtest'] or test_cmd != DEFAULT_TEST_CMD:
-            cmd = ' '.join([
-                self.cfg['pretestopts'],
-                test_cmd,
-                self.cfg['runtest'],
-                self.cfg['testopts'],
-            ])
+        run_test = self.cfg['runtest']
```

Comment:
better use `runtest` to align with the easyconfig parameter name?

Reply:
Makes sense